Makefile

```diff
@@ -1,10 +1,14 @@
+OS       ?= $(shell uname -s)
 CC       ?= gcc
 CXX      ?= g++
 RM       ?= rm -f
 CP       ?= cp -a
-MKDIR    ?= mkdir
+MKDIR    ?= mkdir -p
```

Comment:
I believe there have previously been discussion regard the portability of `-p`.

https://github.com/sass/libsass/issues/1433